src/SFA.DAS.EmployerIncentives.Abstractions/DTOs/Commands/IncentiveLearnerIdentifierDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerIncentives.Abstractions.DTOs.Commands
+{
+    public class IncentiveLearnerIdentifierDto
```

Comment:
We should raise a tech debt to remove the dto suffix and also move these classes out of the abstractions project.

Reply:
Created ticket EI-1715